Cargo.toml

```diff
@@ -12,14 +12,13 @@ keywords = ["lock-free", "rcu", "atomic", "garbage"]
 categories = ["concurrency", "memory-management"]
 
 [features]
-nightly = []
+default = ["std"]
+std = ["lazy_static"]
```

Comment:
Really minor nit: Is there a widely accepted convention for what should this feature be named? It seems the following crates call it `use_std` rather than just `std`:

* [libc](https://github.com/rust-lang/libc/blob/master/Cargo.toml)
* [futures](https://github.com/alexcrichton/futures-rs/blob/master/Cargo.toml)
* [either](https://github.com/bluss/either/blob/master/Cargo.toml#L23)

On the other hand, [failure](https://github.com/withoutboats/failure/blob/master/Cargo.toml) does call it `std`...

Reply:
I don't think there's really a universal convention. It probably doesn't matter since nobody will use that feature name directly: anyone wanting `no_std` will just use `default-features = false`.

Reply:
I noticed that we control whether to use `std` or not with `default-features = false`.  To my eyes `std` and `default` seems quite unrelated..  Is it an accepted convention?

Reply:
Yes, this seems to be a widely used convention. I'm not too happy about it, but there isn't much else that we can do with Cargo.

Also, after looking at a few crates, it seems that `use_std` is actually the standard feature name for rust-lang-nursery crates, so I'll just use that.